BuildAndDeploySteps.md

```diff
@@ -0,0 +1,29 @@
+# BUILDING
+
+## Build Hosting Process
```

Comment:
This file is the main addition. 
Other added files are generated from the build
Some project settings were adjusted to prevent post-deployment issues